docs/installation/compatibility.md

```diff
@@ -1,64 +1,46 @@
-# Ensuring compatibility with legacy frameworks
+# Use GOV.UK Frontend with old frameworks or colours
 
-GOV.UK Frontend has been designed to work alongside the 'legacy frameworks' -
-[GOV.UK Frontend Toolkit], [GOV.UK Template] and [GOV.UK Elements].
+You can configure GOV.UK Frontend to work with the following 'legacy' frameworks:
 
-If you are using components from GOV.UK Frontend within a page that also
-includes any of the legacy frameworks, you should [enable compatibility
-mode](#compatibility-mode).
+- [GOV.UK Frontend toolkit]
+- [GOV.UK Template]
+- [GOV.UK Elements]
 
-If a part of your service uses GOV.UK Frontend without including the legacy
-frameworks, but you need to provide a consistent experience with these
-frameworks for your users, you can configure GOV.UK Frontend to [use the legacy
-colour palette](#using-the-legacy-colour-palette).
+You cannot use the following features if you installed GOV.UK Frontend from `dist`. [Install with node package manager](installing-from-npm.md) instead.
 
-Enabling either of these options means that you are no longer benefiting from
-many of the accessibility and usability improvements made in GOV.UK Frontend, as
-it prioritises providing a consistent experience for the user.
+If you use these features, your service will not have GOV.UK Frontend's many accessibility and usability improvements.
 
-In order to enable either of these options you'll need to have [installed GOV.UK
-Frontend using npm](installing-with-npm.md). These features are not available if
-you are using the 'dist' version.
+## Turn on 'compatibility mode'
 
+You can turn on 'compatibility mode' if you want to use both GOV.UK Frontend components and a legacy framework in a service.
 
-## Compatibility mode
+GOV.UK Frontend components in your pages will:
 
-You can enable compatibility mode for each legacy framework individually.
+- use the old colour palette
+- use the old GOV.UK font from GOV.UK Template
+- override some of the CSS in the legacy frameworks
+- no longer use `rem` for font sizes
 
-You can do this by including configuration in your application's Sass file,
-before you include GOV.UK Frontend:
+To turn on compatibility mode for all 3 legacy frameworks, add the following variables to your project's Sass file, above `@import "govuk-frontend/all"`:
 
 ```SCSS
 $govuk-compatibility-govukfrontendtoolkit: true;
 $govuk-compatibility-govuktemplate: true;
 $govuk-compatibility-govukelements: true;
-
-@import "govuk-frontend/all";
 ```
 
-If compatibility mode is enabled for any framework, GOV.UK Frontend will:
-
-- use the legacy colour palette
-- no longer use rem for font sizes
-- include adjustments to the CSS to 'counter' specific CSS rules in the legacy
-  frameworks
-- use the previous version of the font included with GOV.UK Template
+Set a variable to `false` to turn off compatibility mode for that framework.
```

Comment:
I _think_ we'd generally encourage people to omit the variable entirely if they're not changing it from the default.

Reply:
Thank you - fixed in https://github.com/alphagov/govuk-frontend/pull/1439/commits/f6214966822764e76468e3e5d4b672f9c4796c7f.